libs/common/utilities/src/css/layout/margin/stories/vl-margin.stories-doc.mdx

```diff
@@ -0,0 +1,67 @@
+import {Canvas} from '@storybook/blocks';
+import {Meta} from '@storybook/addon-docs';
+import marginEnPadding from './margin-en-padding.png';
+import * as VlMarginStories from './vl-margin.stories';
+
+<Meta title="Styles-next/Layout (afnemers)/margin"/>
+
+# Margin
+
+## Inhoudstafel
+
+- [Doel](#doel)
+- [Gebruik](#gebruik)
+
+
+## Doel
+
+De CSS-classes om de margin te beïnvloeden zijn gelijkaardig aan deze om de
+[padding](/docs/styles-next-layout-padding--documentatie) te beïnvloeden.
+
+<img src={marginEnPadding} alt="margin en padding" style={{maxWidth: 300 + 'px'}} />
+
+
+## Gebruik
+
+De mogelijkheden zijn bewust beperkt gehouden. Afnemers moeten er naar streven zich te beperken tot de
+hieronder besproken (voorziene) style-classes. Echter, zoals steeds: het is niet verboden om eigen styling
+(en dus een afwijking) te voorzien als dat de layout en bladspiegel ten goede komt.
+
+### vl-margin-next - default
+
+De default marge wordt beïnvloed door de container, typisch wat er voor die resolutie bepaald is door
+[vl-section-next](/docs/styles-next-layout-afnemers-section--documentatie).
+
+<Canvas of={VlMarginStories.MarginDefault} />
+
+### vl-margin-next--small
+
+De margin onderaan en bovenaan wordt op de variabele `--vl-spacing--small` (1.5rem) gezet, links en rechts is er geen
+margin (0), voor kleine schermen (&lt;767px) wordt de `--vl-spacing--normal` (2rem) gebruikt (boven- en onderaan).
+
+<Canvas of={VlMarginStories.MarginSmall} />
+
+### vl-margin-next--medium
+
+De margin onderaan en bovenaan wordt op de variabele `--vl-spacing--medium` (3rem) gezet, links en rechts is er geen
+margin (0), voor kleine schermen (&lt;767px) wordt de `--vl-spacing--normal` (2rem) gebruikt (boven- en onderaan).
+
+<Canvas of={VlMarginStories.MarginMedium} />
+
+### vl-margin-next--no
+
+Nergens margin: alle margin rondom staat op 0.
+
+<Canvas of={VlMarginStories.MarginNo} />
+
+### vl-margin-next--no-bottom
+
+Onderaan geen margin: deze wordt op 0 gezet.
+
```

Comment:
Dit lijkt niet correct:

![image](https://github.com/user-attachments/assets/d0a51696-7b8b-43a1-83bf-25379b0d6b98)

Reply:
Dit is wel correct, zoals vorige commentaar vermeld heeft:

Afgesproken: bij de voorbeelden iets toevoegen in de zin van:

- er is een standaard margin voorzien (`1.5rem` in dit geval)
- die wordt dan overschreven door de bv. `vl-margin-next--small` toe te passen (of desbetreffende padding klasse)

Reply:
aangepast: code via de 'parameters' versimpelt, kleuren / borders aangepast, tekst toegevoegd